src/core/agent/agent.ts

```diff
@@ -218,8 +218,15 @@ class Agent {
         agentUrls.bootUrl
       );
       try {
-        const bootResponse = await this.signifyClient.boot();
-        const bootResponseBody = await bootResponse.json();
+        let bootResponse = await this.signifyClient.boot();
+        let bootResponseBody = await bootResponse.json();
+
+        if (bootResponse.status !== 202) {
+          await new Promise((r) => setTimeout(r, 1000));
+          bootResponse = await this.signifyClient.boot();
+          bootResponseBody = await bootResponse.json();
```

Comment:
This is not what I meant. All we need to do is remove the if statement below that checks for "agent already exists". So if the response is 200 or 202, it's OK. I don't want it to auto re-try if it's 200, that doesn't make sense

Reply:
```
      try {
        await this.signifyClient.boot();
      } catch (e) {
        /* eslint-disable no-console */
        console.error(e);
        throw new Error(Agent.KERIA_BOOT_FAILED);
      }
```

Reply:
Modified